docs/reference/setup/install/docker.asciidoc

```diff
@@ -312,7 +312,16 @@ over the configuration files in the image.
 
 You can set individual {es} configuration parameters using Docker environment variables. 
 The <<docker-compose-file, sample compose file>> and the 
-<<docker-cli-run-dev-mode, single-node example>>  use this method. 
+<<docker-cli-run-dev-mode, single-node example>> use this method.
+
+If you are providing secrets, such as passwords, to {es} and do not want to pass them
+directly via environment variables, you can supply environment variables suffixed with
+"_FILE". The variable value specifies a file whose contents are used for the secret.
+
+For example, if you specified the environment variable "ELASTIC_PASSWORD_FILE" with value
+"/run/secrets/password.txt" and bind-mounted a file at this location in the container with
+the contents "PleaseChangeMe", this would be equivalent to specifying the environment
+variable "ELASTIC_PASSWORD" with the value "PleaseChangeMe".
```

Comment:
In general, I'd recommend sticking to present tense for examples. In this case, I'd go with something like:

> For example, to set the {es} bootstrap password from a file, you can bind mount the file and set the `ELASTIC_PASSWORD_FILE` environment variable to the mount location. If you mount the password file to `/run/secrets/password.txt`, specify:
> 
> `-e ELASTIC_PASSWORD_FILE=/run/secrets/bootstrapPassword.txt \`

Also note that the quotes should be backticks to render the env var and path in code font.